app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryCreateActivity.kt

```diff
@@ -103,58 +97,29 @@ class ItineraryCreateActivity : AppCompatActivity(), OnMapReadyCallback,
         mapboxMap.setStyle(Style.MAPBOX_STREETS) { style ->
             locationComponentManager.enableLocationComponent(style)
 
-            fun onLongClickConsumed(): Boolean {
-                longClickConsumed = true;
-                return false
-            }
-
-            areaBuilderDrawer =
-                MapboxAreaBuilderDrawer(mapView, mapboxMap, style) { onLongClickConsumed() }
-            missionDrawer = MapboxMissionDrawer(mapView, mapboxMap, style)
-
-            mapboxMap.addOnMapClickListener(this)
-            mapboxMap.addOnMapLongClickListener(this)
-
             areaBuilder = PolygonBuilder()
             //areaBuilder = ParallelogramBuilder()
-
-            //areaBuilder.onAreaChanged.add { missionBuilder.withSearchArea(it) }
             areaBuilder.onVerticesChanged.add { areaBuilderDrawer.draw(areaBuilder) }
+            //areaBuilder.onAreaChanged.add { missionBuilder.withSearchArea(it) }
```

Comment:
Are those comments necessary ?

Reply:
I kept them to make explicit how to further adapt the code as soon as we have the necessary implementation.